classes/search/cmsfield.php

```diff
@@ -131,6 +133,26 @@ public function get_document($record, $options = []) {
             $valueformat = $record->valueformat;
         }
 
+        // Add mustache template to value.
+        if (!empty($defaultvalues[$record->typeid]->mustache)) {
+            $cms = new cms($cm->instance);
+            $renderer = new renderer($cms);
+            ob_start();
+            try {
+                // Indexer uses "Empty" session, it may get an error from rendering.
+                $value .= $renderer->get_html();
+            } catch (\Exception $e) {
+                // Use template when an error occurs.
+                $value .= ' ' . $defaultvalues[$record->typeid]->mustache;
```

Comment:
I don't think we should ever index "naked" template

Reply:
I removed try-catch section, so there is no raw template any more.